crates/turbopack-ecmascript-runtime/js/package.json

```diff
@@ -7,10 +7,10 @@
   "scripts": {
     "check": "run-p check:*",
     "check:nodejs": "tsc -p src/nodejs",
-    "check:dev-client": "tsc -p src/dev/client",
-    "check:dev-runtime-base": "tsc -p src/dev/runtime/base",
-    "check:dev-runtime-dom": "tsc -p src/dev/runtime/dom",
-    "check:dev-runtime-edge": "tsc -p src/dev/runtime/edge"
+    "check:browser-client": "tsc -p src/browser/client",
+    "check:browser-runtime-base": "tsc -p src/browser/runtime/base",
```

Comment:
(commenting here to start a comment thread)

Maybe it should be `browser-dev` as the base runtime contains react refresh and hot module reloading code that's only used in dev?

Reply:
Yeah I'm going to split them up further in a follow-up PR. Wanted to keep the changes small as these broke tests a few times when I tried making larger changes to it.

Reply:
I'd rather have the name follow functionality than the other way around (i.e. I'd prefer changing functionality first and then the name)

Reply:
Already working on this change but it's actually more correct to not name it dev currently as it's used for builds as well at this point, that's what I'm actually working on, adding a production runtime

Reply:
Pushed up the changes!